src/components/Chart/Table.js

```diff
@@ -0,0 +1,255 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+
+import { css } from 'glamor'
+import { scaleThreshold, scaleQuantize, scaleOrdinal } from 'd3-scale'
+import { extent } from 'd3-array'
+
+import { useColorContext } from '../Colors/ColorContext'
+import { getFormat, getTextColor } from './utils'
+import { ExpandMoreIcon, ExpandLessIcon } from '../Icons'
+import { defaultProps } from './ChartContext'
+import { sansSerifRegular18 } from '../Typography/styles'
+
+const styles = {
+  container: css({
+    overflowX: 'auto',
+    overflowY: 'hidden',
+    marginLeft: '-20px',
+    marginRight: '-20px'
+  }),
+  table: css({
+    ...sansSerifRegular18,
+    lineHeight: '1.2',
+    minWidth: '100%',
+    borderSpacing: '0 2px',
+    borderCollapse: 'separate',
+    padding: '0 10px'
+  }),
+  header: css({
+    borderBottomWidth: '1px',
+    borderBottomStyle: 'solid',
+    padding: '8px 0',
+    userSelect: 'none',
+    '&:first-of-type': {
+      padding: '8px 0 8px 10px'
+    },
+    '&:last-of-type': {
+      padding: '8px 10px 8px 0'
+    }
+  }),
+  cell: css({
+    padding: '8px 0',
+    '&:first-of-type': {
+      padding: '8px 0 8px 10px'
+    },
+    '&:last-of-type': {
+      padding: '8px 10px 8px 0'
+    }
+  })
+}
+
+const Table = props => {
+  const [colorScheme] = useColorContext()
+  const {
+    values,
+    numberFormat,
+    colorRanges,
+    colorRange,
+    defaultSortColumn,
+    thresholds,
+    tableColumns
+  } = props
+  const columns = values.columns
+
+  const [sortBy, setSortBy] = useState({
+    key: defaultSortColumn,
+    order: 'desc',
+    enableSorting: !!defaultSortColumn || false
```

Comment:
```suggestion
    order: 'desc'
```
The key is and can be set to `undefined` if it should not sort.